### ISSUE TITLE
Scene labels

### ISSUE THREAD

**Issue by Fazlik995** [vt-vl-lab/SDN#2]
@jinwchoi, could you explain more about obtaining pseudo scene labels, please?

I would like to get UCF101 pseudo scene labels

Thank you

**Comment by jinwchoi** [comment]
Hi @Fazlik995 

I just ran ResNet50 pre-trained on Places365 on the Kinetics dataset. Then I got the classifier output vectors and save them all as a numpy file. You can download the pre-trained model here: https://github.com/CSAILVision/places365

**Comment by Fazlik995** [comment]
@jinwchoi,  thank you for replying fast 

I have several questions:

1.which one resnet50_places365.pth.tar or resnet50_places365.t7?
2.Did you ran this pretrained models on Kensho Hara's code, right(just insert above file in --pretrained_path).
3.If so, how did you load pickle format?
